.github/workflows/test.yml

```diff
@@ -30,21 +30,33 @@ jobs:
     - name: Run Memgraph MAGE image
       run: |
         docker run -d -p 7687:7687 memgraph:mage --telemetry-enabled=False
+    - name: Set up C++
```

Comment:
Should we run tests in Docker? Requires 2 Docker images, first for Memgraph query modules, second only with algorithms?

Reply:
Why 2 docker images?